src/many-abci/src/module.rs

```diff
@@ -34,7 +36,9 @@ fn _many_block_from_tendermint_block(block: tendermint::Block) -> Block {
                 id: TransactionIdentifier {
                     hash: hasher.finalize().to_vec(),
                 },
-                content: Some(b),
+                content_: None,
+                request: minicbor::decode(b.as_ref()).ok(),
+                response: None,
```

Comment:
We should probably fill this field

Reply:
Before, there was a single content field, and we are now using that field to populate the request, so what should we put here in the response?